modules/reference/pages/rpk/rpk-cloud/rpk-cloud-auth-use.adoc

```diff
@@ -2,7 +2,7 @@
 
 Select the `rpk` cloud authentication (auth) to use.
 
-This swaps the current cloud authorization to the specified cloud authorization. If your current profile is a cloud profile, this unsets the current profile (because the authorization is now different). If your current profile is for a self-hosted cluster, the profile is kept.
+This swaps the current cloud authentication to the specified cloud authentication. If your current profile is a cloud profile, this unsets the current profile (because the authorization is now different). If your current profile is for a self-hosted cluster, the profile is kept.
```

Comment:
```suggestion
This swaps the current cloud authentication to the specified cloud authentication. If your current profile is a cloud profile, this resets the current profile (because the authorization is now different). If your current profile is for a self-hosted cluster, the profile is kept.
```

Reply:
"unsets" is unclear. Just not sure what that means here. What happens when the profile is "unset"? It reverts to ...what? Thinking the correct word here might be reset...if wrong, let's discuss what the actual behavior here is so we can get it right. Thx.

Reply:
rpk always uses a "selected" profile when talking to a cluster, if no profile is set/selected, then rpk uses the defaults (localhost:<default-port>)

`unsets` here means that rpk won't have a profile selected, and the user would have to either create a new one (which automatically sets that as the 'selected profile') or run:

```
rpk profile use [profile name you want to use]
```

Reply:
Keeping unsets, as it better reflects the behavior